service/controller/v9/version_bundle.go

```diff
@@ -32,6 +32,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Add VNET service endpoint for accessing Azure Storage.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "kubernetes",
+				Description: "Update kubernetes to 1.14.5 (CVE-2019-11247) https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1145",
+				Kind:        versionbundle.KindChanged,
```

Comment:
This PR only added a changelog about the k8s version but not about the LB. Can someone take care or should I provide a PR?